apps/www/components/Frame/index.js

```diff
@@ -104,16 +101,16 @@ const Frame = ({
   formatColor,
   footer = true,
   pullable,
-  isMember,
   hasOverviewNav: wantOverviewNav,
   stickySecondaryNav,
   isOnMarketingPage,
   pageColorSchemeKey,
 }) => {
   const { inNativeApp, inNativeAppLegacy } = useInNativeApp()
-  const { me } = useMe()
+  const { t } = useTranslation()
+  const { me, hasActiveMembership } = useMe()
 
-  const hasOverviewNav = isMember && wantOverviewNav
+  const hasOverviewNav = hasActiveMembership && wantOverviewNav
```

Comment:
`hasActiveMembership` is not the right thing here, should be `hasAccess` instead (isMember = has role member = hasAccess – includes access grants, hasActiveMembership = is paid member)